### ISSUE TITLE
The Screen

### ISSUE THREAD

**Issue by nth-erik** [rangeoshun/wormer#36]
_What is [**this**](https://www.youtube.com/watch?v=7ffj8SHrbk0)?_

**Comment by rangeoshun** [comment]
Please ellaborate!

**Comment by nth-erik** [comment]
To be absolutely honest about this crucial yet seemingly marginary user-experience issue, there seems to be an overwhelmingly high resolution for an underwhelmingly subvolumetric arena, in which precise movement-coordination and decision-making are to be expected at an intense pace, in which regards it would be of utmost importance to be able to visually predetermine the linear and non-linear path taken by the player's ever-expanding, serpentine avatar entity.

**TL;DR**
The screen is way too tiny on a desktop screen resolution. And no, `Ctrl + Mouse Wheel` isn't the right solution to that problem. ;-)

**Comment by rangeoshun** [comment]
Git the wrong issue ID in commit, but bigger screen is done :)
https://github.com/rangeoshun/wrom/commit/ef3a6d30fdb9bbdc099cd1094d3a7d2db18449b7

**Comment by nth-erik** [comment]
Thank you! This one is done. :)
